hack/deploy-to-nodes.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+source hack/common.sh
+source cluster-up/cluster/$KUBEVIRT_PROVIDER/provider.sh
+
+set -e
+
+readonly PODMAN_SOCKET=${PODMAN_SOCKET:-"/run/podman/podman.sock"}
+
+detect_podman_socket() {
+    if curl --unix-socket "${PODMAN_SOCKET}" http://d/v3.0.0/libpod/info >/dev/null 2>&1; then
+        echo "${PODMAN_SOCKET}"
+    fi
+}
+
+if [ "${CONTAINER_CLIENT}" = "podman" ]; then
+    _cri_bin="podman --remote --url=unix://$(detect_podman_socket)"
+elif [ "${CONTAINER_CLIENT}" = "docker" ]; then
+    _cri_bin=docker
+else
+    _cri_socket=$(detect_podman_socket)
+    if [ -n "$_cri_socket" ]; then
+        _cri_bin="podman --remote --url=unix://$_cri_socket"
+        echo >&2 "selecting podman as container runtime"
+    elif docker ps >/dev/null 2>&1; then
+        _cri_bin=docker
+        echo >&2 "selecting docker as container runtime"
+    else
+        echo >&2 "no working container runtime found. Neither docker nor podman seems to work."
+        exit 1
+    fi
+fi
```

Comment:
Similar logic exist in cluster-up/ephemeral-provider-common.sh, the fn name is `detect_podman_socket`.
Is it possible to reuse it?

Reply:
OK, make sense, lets do it on another PR.
Thanks for explaining.